TESTS/psa/crypto_init/main.cpp

```diff
@@ -16,14 +16,15 @@
 * limitations under the License.
 */
 
-#if ((!defined(TARGET_PSA)) || (!defined(MBEDTLS_PSA_CRYPTO_C)))
-#error [NOT_SUPPORTED] Mbed Crypto is OFF - skipping.
+#if ((!defined(TARGET_PSA)) || (!defined(MBEDTLS_PSA_CRYPTO_C)) || (!defined(MBEDTLS_PSA_CRYPTO_SPM )))
+#error [NOT_SUPPORTED] Mbed SPM Crypto is OFF - skipping.
```

Comment:
Why can't we run tests without SPM enabled? Mbed Crypto should work fine without SPM.

Reply:
OK, this is just for the crypto init test.